### PR TITLE
Added hint how fosjsrouting works

### PR DESCRIPTION
**Pull request by marek5816**
| Q | A
| --- | ---
| Fixed tickets | https://github.com/sulu/sulu-docs/issues/750
| Related PRs | https://github.com/sulu/sulu-docs/pull/757
| License | MIT

#### What's in this PR?

Added hints according to https://github.com/sulu/sulu-docs/issues/750

Originally created here (accidentally closed) https://github.com/sulu/sulu-docs/pull/757

### CODE REVIEW

**Comment by marek5816** [comment]
@alexander-schranz Any update/suggestions on this?

File: book/extend-admin.rst [hunk 1]
```diff
@@ -293,6 +293,12 @@ as well, then you should be able to see these actions when using the ``debug:rou
       app.put_event      PUT      ANY      ANY    /admin/api/events/{id}.{_format}
       app.delete_event   DELETE   ANY      ANY    /admin/api/events/{id}.{_format}
 
+.. note::
+
+    Routes to this actions must be exposed using `FOSJsRoutingBundle`_, and also they must have set ``methods``
+    parameter (e.g. ``options: ['expose' => true], methods: ['GET']``). Examples can be found at `Sulu Workshops`_, specificly in assignment 10.
```
Comment (Prokyonn) [inline book/extend-admin.rst]:
```suggestion
    The routes to these actions must be exposed using `FOSJsRoutingBundle`_, and also they must have set the ``methods``
    parameter (e.g. ``options: ['expose' => true], methods: ['GET']``). Examples can be found in the `Sulu Workshops`_, specifically in assignment 10.
```

@alexander-schranz WDYT does this information already suffice or do you want it in more detail?

**Review by Prokyonn — APPROVED** [review]
Looks good from my side, @alexander-schranz please have a final look :)

**Review by alexander-schranz — APPROVED** [review]

**Comment by alexander-schranz** [comment]
@marek5816 thank you!

**Pull request merged**